ckanext/recombinant/templates/datatables/datatables_view.html

```diff
@@ -0,0 +1,70 @@
+{% ckan_extends %}
+
+{%- block datatable_field_headers -%}
+
+  {% set chromo = h.recombinant_published_resource_chromo(resource.id) %}
+  {% if chromo %}
+
+    {% set chromo_keyed = {} %}
+    {% for recombinant_field in chromo.fields %}
+      {% set _ = chromo_keyed.update({recombinant_field.datastore_id: {'description': recombinant_field.description,
```

Comment:
definitely don't want to replace the gettext function with None
```suggestion
      {% do chromo_keyed.update({recombinant_field.datastore_id: {'description': recombinant_field.description,
```

Reply:
W var name BTW